tests/service/data/SyntaxTree/AbstractKeyword.fs.bsl

```diff
@@ -0,0 +1,45 @@
+ImplFile
+  (ParsedImplFileInput
+     ("/root/AbstractKeyword.fs", false, QualifiedNameOfFile AbstractKeyword, [],
+      [],
+      [SynModuleOrNamespace
+         ([AbstractKeyword], false, AnonModule,
+          [Types
+             ([SynTypeDefn
+                 (SynComponentInfo
+                    ([], None, [], [X],
+                     PreXmlDoc ((1,0), FSharp.Compiler.Xml.XmlDocCollector),
+                     false, None, /root/AbstractKeyword.fs (1,5--1,6)),
+                  ObjectModel
+                    (Unspecified,
+                     [AbstractSlot
+                        (SynValSig
+                           ([], SynIdent (Y, None),
+                            SynValTyparDecls (None, true),
+                            LongIdent (SynLongIdent ([int], [], [None])),
+                            SynValInfo ([], SynArgInfo ([], false, None)), false,
+                            false,
+                            PreXmlDoc ((2,4), FSharp.Compiler.Xml.XmlDocCollector),
+                            None, None, /root/AbstractKeyword.fs (2,4--2,20),
+                            { LeadingKeyword =
+                               Abstract /root/AbstractKeyword.fs (2,4--2,12)
+                              InlineKeyword = None
+                              WithKeyword = None
+                              EqualsRange = None }),
+                         { IsInstance = true
+                           IsDispatchSlot = true
+                           IsOverrideOrExplicitImpl = false
+                           IsFinal = false
+                           GetterOrSetterIsCompilerGenerated = false
+                           MemberKind = PropertyGet },
+                         /root/AbstractKeyword.fs (2,4--2,20),
+                         { GetSetKeywords = None })],
+                     /root/AbstractKeyword.fs (2,4--2,20)), [], None,
+                  /root/AbstractKeyword.fs (1,5--2,20),
+                  { LeadingKeyword = Type /root/AbstractKeyword.fs (1,0--1,4)
+                    EqualsRange = Some /root/AbstractKeyword.fs (1,7--1,8)
+                    WithKeyword = None })], /root/AbstractKeyword.fs (1,0--2,20))],
+          PreXmlDocEmpty, [], None, /root/AbstractKeyword.fs (1,0--2,20),
+          { LeadingKeyword = None })], (true, false),
+      { ConditionalDirectives = []
+        CodeComments = [] }, set []))
```

Comment:
Also would be nice to add a new line at the end, so GitHub doesn't add this mark. 🙂